src/Block/Exception/ExceptionLogger.php

```diff
@@ -38,8 +38,12 @@ public function withException(
             return;
```

Comment:
This one can be removed, this separates the options.

Reply:
has been removed :)